frontend/src/libs/components/input/styles.module.scss

```diff
@@ -0,0 +1,28 @@
+.input {
+  display: flex;
```

Comment:
I'm not sure why do we need `display: flex` and `flex-direction` on `input` element, I believe it doesn't change anything, so I suggest removing those css properties